src/modules/27-platform/certificates/pages/certificates/CertificatesPage.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { useParams } from 'react-router-dom'
+
+import type { ModulePathParams, ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import { NGBreadcrumbs } from '@common/components/NGBreadcrumbs/NGBreadcrumbs'
+import ScopedTitle from '@common/components/Title/ScopedTitle'
+import { Scope } from '@common/interfaces/SecretsInterface'
+import { getLinkForAccountResources } from '@common/utils/BreadcrumbUtils'
+import { Page } from '@common/exports'
+import { useStrings } from 'framework/strings'
+
+const CertificatesPage: React.FC<Record<string, unknown>> = () => {
```

Comment:
Looks like this doesn't take props, do you need the typing?
```suggestion
const CertificatesPage: React.FC = () => {
```

Reply:
Not right now, maybe in subsequent PRs.